### PR TITLE
Popout window project

### PR DESCRIPTION
**Pull request by Abbas-Aljuaifari**
Previously added ->

Full desktop version
Add transitions and animations
Add Deployment
When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
Form validation by using DOM, JavaScript and CSS

For those used the following tools ->

CSS Flexbox
CSS Grid
HTML Semantics
HTML Box Model
HTML Form etc.
CSS Transitions
CSS Animations
GITHUB Page
JavaScript
DOM Manipulation

In this branch add ->

When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: script.js
Comment (Efiamotu-1) [inline script.js:1]: - You have done a great job adding the projects dynamically using javascript and creating an array to store project details howver you are missing a key requiremtn which is stated in the image below.
d-q
- [ ] While your project has a modal popup, you are currently displaying the same popup for every project but this shouldn't be the case, you are to dynamically display the correct popup for every project that is selected with the information provided in the modal array of objects you created. It is important to it this way becasue when you finally decide to insert future project which you will work on in your portfolio you will not want a static information to be used for each project rather you will want the correct information about the project to be displayed such as the images, links and description.
- please kindly ensure before resubmitting each project once clicked is displaying unique contents and links as you have provided for them in your projects array on `line 85` :+1: :sparkles: 
- Also ensure you follow all of the design pattern for the modal on both destop and mobile version as provided by the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Abbas-Aljuaifari , 



Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js
Comment (Munsa1) [reply script.js:1]: - [ ] As requested by the previous reviewer, kindly ensure  you follow the given design pattern, this includes the position of the heading and the close `X` icon for the modal in mobile view. kindly fix this.

Given design
image

Your design
image
- [ ] Also ensure the bottom buttons are in view by adding  some padding and marginn to the bottom of the buttons. Dont worry if the modal becomes longer in the vertical view, add the `overfolw-y: auto;` PROPERTY to add a scroll effect

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Abbas-Aljuaifari, 
required-changes-1

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive PR :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Abbas-Aljuaifari** [comment]
Hi here is how it shows on my inspection tool, I  don't understand why it shows like that on your end.
Screenshot 2023-05-20 at 17 53 01

**Review by taldr27 — APPROVED** [review]
# Status: Approved! 🟢 
Hello team @Abbas-Aljuaifari , @JansueT  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉
✅ Professional and customized README. 🤓
✅ Linters tests passing. 👍
✅ Incorporates feedback from previous code reviews. 💯
✅ Populating the popup dynamically. 🎯
✅ Popup functionality working fine. 🙌

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Pull request merged**